test/blackbox-tests/test-cases/dune-init.t/run.t

```diff
@@ -251,24 +251,23 @@ Comments in dune files are preserved
 Will not create components with invalid names
 
   $ dune init lib invalid-component-name ./_test_lib
```

Comment:
```suggestion
  $ dune init lib invalid-component-name ./_test_lib
```
Should these be in code block or monospace. I think it's up to you, as it's clearly something for the command line.

Reply:
This is cram syntax, this is interpreted.